lib/kube_link_generator.rb

```diff
@@ -83,7 +83,7 @@ def get_pod_instance_info(pod, job, pods_per_image)
       'index' => index,
       'id' => pod.metadata.name,
       'az' => pod.metadata.annotations['failure-domain.beta.kubernetes.io/zone'] || 'az0',
-      'address' => pod.status.podIP,
+      'address' => pod.metadata.name + '.' + pod.spec.subdomain + '.' + "#{ENV['KUBERNETES_NAMESPACE']}" + '.' + "#{ENV['KUBERNETES_CLUSTER_DOMAIN']}",
```

Comment:
This may be easier to read as `"#{pod.metadata.name}.#{pod.spec.subdomain}.#{ENV['KUBERNETES_NAMESPACE']}.#{ENV['KUBERNETES_CLUSTER_DOMAIN']}"`.

Reply:
@mook-as 
Thank you for your review. I'll update and add spec test cases in next commit. Thanks!

Reply:
Hi @mook-as,

All comments addressed. I noticed there's already a test case for it, so I updated that one. Would you please take a look? Thanks!